src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -695,7 +713,19 @@ public DescriptorImpl() {
             if (repoJobs == null) {
                 repoJobs = new ConcurrentHashMap<String, Set<AbstractProject<?, ?>>>();
             }
-            save();
+            saveAfterPause();
+        }
+        
+        private void saveAfterPause() {
```

Comment:
Haven't seen that in the main installations, though there are less restarts.  I think that's probably worth a bug report on the main Jenkins JIRA in case it's something specific to the version you're running.